docs/pack-command.md

```diff
@@ -0,0 +1,24 @@
+# Pack Command
+
+The Crossplane `pack` command is used to package objects defined in Kubernetes
+manifests into a Crossplane `KubernetesApplication` object by embedding the
+objects as `KubernetesApplicationResourceTemplates`. This allows a grouping of
+resources to be scheduled from a Crossplane Kubernetes control cluster to a
+remote Kubernetes cluster that it has access to.
+
+## Usage
+
+After installation, the `pack` command can be invoked by running `kubectl
+crossplane pack`. The command is commonly used by piping the output from a
+previous command to it. For example, templating a [Helm](https://helm.sh/)
+chart:
+
+```
+helm template crossplane -n crossplane-system crossplane/crossplane-alpha | kubectl crossplane pack -
```

Comment:
have you confirmed that crossplane installed in the remote cluster works? Also it might be more helpful if we were to pack a well-known app's helm chart to show that this is for any app, not Crossplane.

Reply:
I'll add docs for linkerd also :)

Reply:
Actually on second thought I like that we use Crossplane because we are aware if the helm chart changes. In addition, the line after this demonstrates usage with any config.

Reply:
sounds good!